### PR TITLE
Run restorecon if selinux enabled

### LINKED ISSUE #15: SELINUX, labels
Upon using the ansible-role-omero-web on a RHEL system with SELINUX set to enforcing, the static files served up by NGINX are not loading after a fresh installation, or an upgrade of an existing running server.

Note: in the case of the output here, this was already done for the existing server at first deployment, and these steps were re-ran for an upgrade to the existing server.

`[root@pub-omero ~]# audit2why < /var/log/audit/audit.log` shows:

```
type=AVC msg=audit(1522679845.947:375427): avc:  denied  { read } for  pid=20627 comm="nginx" name="OMERO.web" dev="dm-7" ino=726 scontext=system_u:system_r:httpd_t:s0 tcontext=unconfined_u:object_r:unlabeled_
t:s0 tclass=lnk_file

        Was caused by:
                Missing type enforcement (TE) allow rule.

                You can use audit2allow to generate a loadable module to allow this access.
```
after creating an audit2allow rule as it suggests, the following is generated:

```
[root@ ~]# audit2allow < /var/log/audit/audit.log


#============= abrt_t ==============
allow abrt_t inetd_child_t:dbus send_msg;

#============= httpd_t ==============

#!!!! WARNING: 'unlabeled_t' is a base type.
#!!!! The file '/opt/omero/web/OMERO.web' is mislabeled on your system.
#!!!! Fix with $ restorecon -R -v /opt/omero/web/OMERO.web
allow httpd_t unlabeled_t:lnk_file read;
```

After running `restorecon -R -v /opt/omero/web/OMERO.web`, the statics are happily served without a restart of NGINX.

### PR DESCRIPTION
**Pull request by manics**
Fixes #15 
Replacement for https://github.com/ome/ansible-role-omero-web/pull/19

This should also work as expected in check-mode (give a status output without making changes). Untested.

### CODE REVIEW

**Comment by manics** [comment]
This was used to deploy https://github.com/ome/prod-playbooks/pull/186 without manual intervention

**Comment by sbesson** [comment]
Looks good from a code-review perspective and hopefully should reduce manual steps when upgrading a Web deployment with SELinux enabled.

I will have a go at testing the downgrade/upgrade process using `pub-omero` later this week. If this passes, this should be ready to merge and release.

**Comment by sbesson** [comment]
After a couple tries, I failed to reproduce the issue #15 on `pub-omero` while downgrading/upgrading OMERO.web in isolation. I'll propose this a try within a context of a full deployment and get it merged and tagged as a patch release of the role.

File: tasks/web-install.yml [hunk 1]
```diff
@@ -126,3 +126,13 @@
...
   - omero-web restart omero-web
+
+- name: omero web | selinux restorecon
+  become: yes
+  command: >-
+    /usr/sbin/restorecon -R -v
+    {{ ansible_check_mode | ternary('-n', '') }}
+    {{ omero_web_basedir }}/OMERO.web
+  check_mode: no
+  changed_when: 'result.stdout != ""'
```
Comment (sbesson) [inline tasks/web-install.yml]:
```
TASK [ome.omero_web : omero web | selinux restorecon] ********************************************************************
skipping: [ome-demoserver.openmicroscopy.org]
fatal: [pub-omero.openmicroscopy.org]: FAILED! => {"msg": "The conditional check 'result.stdout != \"\"' failed. The error was: error while evaluating conditional (result.stdout != \"\"): 'result' is undefined"}
```

**Comment by sbesson** [comment]
Successfully tested in and outside check-mode with https://github.com/ome/prod-playbooks/pull/192

**Pull request merged**